Seeq.Link.SDK.Debugging.Agent/EntryPoint.cs

```diff
@@ -14,9 +18,28 @@ namespace Seeq.Link.Debugging.Agent {
     public class EntryPoint {
 
         public static void Main(string[] args) {
-            log4net.Config.XmlConfigurator.Configure();
+            XmlConfigurator.Configure();
 
-            Seeq.Link.Agent.Program.Configuration config = Seeq.Link.Agent.Program.GetDefaultConfiguration();
+            const string agentName = ".NET Connector SDK Debugging Agent";
+            var executingAssemblyLocation = Assembly.GetExecutingAssembly().Location;
+            var seeqDataFolder = Path.Combine(Path.GetDirectoryName(executingAssemblyLocation), "data");
+
+            if (AgentKeyHelper.IsAgentOneTimePasswordSet()) {
```

Comment:
One idea (take it of leave it): How about pushing the whole block of code into AgentKeyHelper and call only something like AgentKeyHelper.SetupOtp(agentName, seeqDataFolder)?

I'd prefer to be less verbose in the EntryPoint.cs if possible.

Reply:
it looks great! Thank you